convert2rhel/data/8/x86_64/repos/oracle-8.4/uek-ol8.repo

```diff
@@ -0,0 +1,13 @@
+[ol8_UEKR6]
```

Comment:
Remove this whole file for these reasons:
1) We require users to install and boot into the RHEL-compatible kernel, not UEK, so the UEK doesn't need to be updated
2) the UEK repo is not OL 8.4-specific. It might or might not require dependencies from OL 8.5+ BaseOS/Appstream